param/low_level_motion_sim.yaml

```diff
@@ -19,26 +19,22 @@ roll_d: 0.0
 roll_accumulator_max: 2.0
 roll_accumulator_min: -2.0
 
-yaw_p: -1.0
-yaw_i: 0.0
-yaw_d: 0.0
-yaw_accumulator_max: 0.0
-yaw_accumulator_min: 0.0
-
 throttle_max      : 1.0
 throttle_min      : 0.0
 throttle_max_rate : 1000.0
 
-pitch_max         : 3.5
-pitch_min         : -3.5
-pitch_max_rate    : 1000.0
+pitch_max         : 0.25
+pitch_min         : -0.25
+pitch_max_rate    : 0.25
 
-roll_max          : 3.5
-roll_min          : -3.5
-roll_max_rate     : 1000.0
+roll_max          : 0.25
+roll_min          : -0.25
+roll_max_rate     : 0.25
 
 yaw_max           : 3.5
 yaw_min           : -3.5
 yaw_max_rate      : 1000.0
 
 update_frequency: 60.0
+max_initial_wait_seconds: 10.0
```

Comment:
I think these could use comments explaining what they are for. Perhaps more expressive names too.